From 85f56e22785e4ff2a97e06a7f26a0c48fccd782d Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 24 Jun 2019 10:47:11 +0800
Subject: [PATCH] input: touchscreen: gt9xx: avoid clang warning

drivers/input/touchscreen/gt9xx/gt9xx_update.c:2744:19: warning: equality comparison with extraneous parentheses [-Wparentheses-equality]
 while ((ready == 0)) //Wait for measurement complete
         ~~~~~~^~~~

Change-Id: I6954c7cd8867e1d85d435a23ee857cd4e3c58b4b
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/input/touchscreen/gt9xx/gt9xx_update.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/gt9xx/gt9xx_update.c b/drivers/input/touchscreen/gt9xx/gt9xx_update.c
index 974d8d829d1a..0cfbf11b9a51 100644
--- a/drivers/input/touchscreen/gt9xx/gt9xx_update.c
+++ b/drivers/input/touchscreen/gt9xx/gt9xx_update.c
@@ -2741,7 +2741,7 @@ static u32 gup_clk_count_get(void)
     s32 temp;
     s8  buf[4];
 
-    while ((ready == 0)) //Wait for measurement complete
+    while (ready == 0) //Wait for measurement complete
     {
         i2c_read_bytes(i2c_connect_client, _bRO_MISCTL__MEA_RDY, buf, 1);
         ready = buf[0];
-- 
2.35.3

